src/frontend/components/injector-tree/injector-tree.ts

```diff
@@ -63,7 +63,7 @@ export class InjectorTree implements OnChanges {
   private unFocusNode = () => {
     this.focusedComponent = -1;
     this.focusedDependency = -1;
-  };
+  }
```

Comment:
@rajinder-yadav I don't mind either way (though I don't use semicolons on class member declarations myself), but since every other instance in this project uses the not-on-class-memeber-decs convention, I would rather have this instance match them (and the current tslint config).

As far as what the convention should actually be, I'm open to changing it, but we should probably defer to some Rangle standard. At any rate it would have to be done in a separate change obviously.